Bricks/sync/owned_borrowed.h

```diff
@@ -0,0 +1,492 @@
+/*******************************************************************************
+The MIT License (MIT)
+
+Copyright (c) 2016 Dmitry "Dima" Korolev, <dmitry.korolev@gmail.com>.
```

Comment:
Well, it was 2016 when I `git mv`-ed the file. I prefer to stick with the philosophy of keeping the timestamp of the original code.